packages/storybook8/stories/Concepts/Transfer/Docs.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+import TransferSnippetText from '!!raw-loader!./snippets/Transfer.snippet.tsx';
+
+<Meta title="Concepts/Transfer" />
+
+# Transfer
+
```

Comment:
Missing sub heading `overview`

Reply:
I'd intentionally removed it to match the other concepts pages